include/zephyr/acpi/acpi.h

```diff
@@ -0,0 +1,130 @@
+/*
+ * Copyright (c) 2023 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DRIVERS_ACPI_H_
+#define ZEPHYR_INCLUDE_DRIVERS_ACPI_H_
+#include <acpi.h>
```

Comment:
Having Zephyr `acpi.h` include another `acpi.h` is pretty confusing. There might also be the risk for infinite recursion if the referenced file ends up being this same file. I believe the reference is to the `acpi.h` header file in the ACPICA tree, correct? Would it be possible to set the include lookup paths in CMakeLists.txt in such a way that you need to provide some name pacing for the include statement here, e.g. `#include <acpica/acpi.h>`?